app/js/actions/labOrders/labOrderableAction.js

```diff
@@ -0,0 +1,34 @@
+import * as types from '../actionTypes';
+import axiosInstance from '../../config';
+import loading from '../loading';
+
+export const fetchLabOrderablesSuccess = orderables => ({
+  type: types.FETCH_LAB_ORDERABLES_SUCCESS,
+  orderables,
+});
+
+export const fetchLabOrderablesFailure = error => ({
+  type: types.FETCH_LAB_ORDERABLES_FAILURE,
+  error,
+});
+
+/**
+ * @function
+ * @returns {function} - Object consisting lab orderables from API resource
+ * @throws {string} - when an error occurs
+ */
+export const getLabOrderable = () => (dispatch) => {
+  dispatch(loading('FETCH_LAB_ORDERABLES', true));
+  return axiosInstance.get('/concept/da006137-88ca-4d11-ae58-8b4b439afdd6?v=full')
```

Comment:
If there isn't already, let's create a top-level constants.js file and move this uuid there and reference by variable here... this is a concept users will need to make sure they have defined in their system so it will be important for all these requirements to be specified in a single file so a user can easily see what they need to set up.

(Long-term this actually should be referenced by a concept mapping, or we should get this from a global property, but I will ticket that in a separate ticket)

Reply:
No problem @mogoodrich

Reply:
Changes have been made @mogoodrich